src/fontra/views/editor/panel-user-settings.js

```diff
@@ -68,7 +69,22 @@ export default class UserSettingsPanel extends Panel {
     });
 
     items.push({
-      displayName: "Experimental features",
+      displayName: translate("sidebar.user-settings.language"),
+      controller: languageController,
+      descriptions: [
+        {
+          key: "language",
+          ui: "radio",
```

Comment:
I would prefer this to be a popup menu, but I guess ui-form doesn't support that yet. May be a follow-up refinement.